pkg/apis/openstack/v1alpha1/types.go

```diff
@@ -46,6 +46,8 @@ type MachineProviderConfigSpec struct {
 	PodNetworkCidr string `json:"podNetworkCidr"`
 	// The size of the root disk used for the instance.
 	RootDiskSize int `json:"rootDiskSize,omitempty"` // in GB
+	// The type of the root disk used for the instance.
```

Comment:
If this is an optional field then do you think it would be a good idea to add the // +optional tags to these attributes? Even for other fields in the CRD.